LivrCommonCustomRules/LivrCommonCustomRules/CustomRules/ExtraRules.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  ExtraRules.swift
+//  LivrCommonCustomRules
+//
+//  Created by Felipe Marino on 03/07/19.
+//  Copyright © 2019 Zap SA Internet. All rights reserved.
+//
+
+import Foundation
+import Livr
+
+struct CustomRules {
+    
+    struct GreaterThanOrEqualToField: LivrRule, PreDefinedRule, RuleThatCompareFields, CustomRule {
+        var name: String = "greater_eq_than_field"
+        static var name = "greater_eq_than_field"
```

Comment:
Why do you create two variables names with same value?

Reply:
Nice catch. I did wrongly conform to PreDefinedRule, which is not the case. Fixed already.